### ISSUE TITLE
Change to unogenerator to generate odt file

### ISSUE THREAD

**Issue by ole-tange** [turulomio/pdf2odt#19]
On a dir with many small PDF files run:
~~~
parallel -j1 --tag pdf2odt --pdf {} {.}.odt ::: *pdf
~~~

This works fine.

But if you run them in parallel:
~~~
parallel -j10 --tag pdf2odt --pdf {} {.}.odt ::: *pdf
~~~
you sometimes get:
~~~
my.pdf     OSError: [Errno 9] Bad file descriptor
~~~

This looks like a race condition.

**Comment by turulomio** [comment]
pdf2odt uses odfpy to generate libreoffice documents. Unfortunately this library has not been updated for a long time.

pdf2odt should be migrated to [unogenerator](https://github.com/turulomio/unogenerator/) for document generation, which allows parallelism to generate documents

**Comment by turulomio** [comment]
Fixed in pdf2odt-1.0.0
